internal/addrs/module_call.go

```diff
@@ -176,6 +176,15 @@ func (m ModuleCallOutput) UniqueKey() UniqueKey {
 
 func (m ModuleCallOutput) uniqueKeySigil() {}
 
+func (m ModuleCallOutput) ConfigOutputValue() ConfigOutputValue {
+	return ConfigOutputValue{
+		Module: Module{m.Call.Name},
```

Comment:
I don't think you can do this translation from this part of the code. Or you need to pass in the module.

for this module call `module "mymodule" {...}`, `m.Call.Name` is `mymodule` not the absolute path to the current module that `ConfigOutputValue` wants.

You need to either pass in the parent module as an argument here and merge it with the call. Or do this from AbsModuleCall and pass in the output name. Or make a new AbsModuleCallOutput that has the full path to the parent module, the call name, and the output name.